docs/quickstarts/outlook-quickstart.md

```diff
@@ -91,12 +91,12 @@ The add-in project that you've created with the Yeoman generator contains sample
 ### Try it out
 
 > [!NOTE]
-> Office Add-ins should use HTTPS, not HTTP, even when you are developing. If you are prompted to install a certificate after you run the following command, accept the prompt to install the certificate that the Yeoman generator provides.
+> Office Add-ins should use HTTPS, not HTTP, even when you are developing. If you are prompted to install a certificate after you run the following command, accept the prompt to install the certificate that the Yeoman generator provides. You may also have to run your command prompt or terminal as an adminstrator for the changes to be made.
 
 1. Run the following command in the root directory of your project. When you run this command, the local web server will start (if it's not already running).
 
     ```command&nbsp;line
-    npm start
+    npm run start:web
```

Comment:
Why not just npm run dev-server like we did with the Project documentation?

Reply:
Won't this be the command for sideloading online eventually...? Since we plan to have sideloading online for both Outlook and OneNote, I felt it was okay to use this (we already use this in the OneNote docs and it does the same thing as npm run dev-server plus it checks for a certificate and the loopback exemption)

Reply:
I think what we are trying to avoid is having users run "npm start" for Desktop, as sideloading is not supported for some hosts in Desktop, and they will get an error.  This documentation appears to be catered for Desktop, so I don't think we want to use "start:web" here as that command is targeted for Office Online, although it too starts the dev-server. I think just change this to "npm run dev-server"

Reply:
Is this correct?
```suggestion
    npm run dev-server
```

Reply:
@nokafor / @TCourtneyOwen : We had recently changed this from `npm run dev-server` to `npm start` due to this [customer-created PR](https://github.com/OfficeDev/office-js-docs-pr/pull/2068/files). Will we be back to the issue that led to this change in the first place?

Reply:
We did very recently change our Yo Office post-install steps to tell users to run npm run dev-server if on hosts that don't support sideloading.  See the _postInstallHints method at https://raw.githubusercontent.com/OfficeDev/generator-office/master/src/app/index.ts

Reply:
Ok, good to know. I'll sign off on this change but I'll need to go through the other Outlook content to replace `npm start`.